api/contrib/envoy/extensions/compression/compressor/igzip/v3alpha/igzip.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package envoy.extensions.compression.compressor.igzip.v3alpha;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.compression.compressor.igzip.v3alpha";
+option java_outer_classname = "IgzipProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/compression/compressor/igzip/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Igzip Compressor]
+// [#extension: envoy.compression.compressor.igzip]
+
+message Igzip {
+  enum CompressionLevel {
```

Comment:
should we add  next-free-field Notes in here? like `// [#next-free-field: 5]`

Reply:
It is a lint design, with no requirement for less than 6.

Reply:
> It is a lint design, with no requirement for less than 6.

Yeah, using `fix_proto_format` will fix this automatically.

Reply:
Ok, I see, thanks.